src/components/profile/AccessLabel.vue

```diff
@@ -69,12 +101,23 @@ export default {
           return this.fluent('classification_info');
       }
     },
+    isVolunteerWithSlackAccess() {
+      return (
+        !this.staffInformation.staff.value &&
+        'slack-access' in (this.accessInformation.mozilliansorg.values || {})
+      );
+    },
   },
 };
 </script>
 
 <style>
-.data-classification {
+.data-access-label:not(:last-child) {
+  margin-right: 0.3em;
+}
+
+.data-classification,
+.data-slack-access {
```

Comment:
I'd rename this to something like `access-label` (which make the above rule simpler, too / you don't need an extra class anymore).
And then have the modifiers `access-label--data-staff`, `acces-label--data-confidential`, `access-label--slack`.

Reply:
Just asking: With or without the `data-` class-name prefix? I'm unsure about this.

Reply:
this specific class I would replace with just on class called `access-label` without `data-`